src/algs.rs

```diff
@@ -101,7 +101,10 @@ pub fn ipc_valid(v: String) -> std::result::Result<(), String> {
 /// }
 ///
 /// fn main() {
-///     portus::start!("unix", None, MyCongestionControlAlgorithm(Default::default()));
+///     let handle = portus::spawn!("unix", None, MyCongestionControlAlgorithm(Default::default()), 4u32);
```

Comment:
So before this doctest worked because install datapath programs would fail and so the function would exit. But after I added the code to wait if the datapath is not available, this doctest will hang forever. So I think our only options are (a) don't run the doctest, which is a little unfortunate because it's a nice test of the entire API with a sample congestion control algorithm or (b) use the spawn api and have it start ccp and then kill it with the handle. Do you have any other suggestions for being able to still test start too?